Systems/engine.xml

```diff
@@ -528,14 +534,38 @@
                         <value>0.2</value>
                     </greater-than>
                 </and>
-                
-                <!-- engine is also killed if oil level is critical. Minimum oil level in the sump for safe operation (according 
-                to Lycommings manual, page 3-10) is 2 quarts, which explains the value of 1.97 below-->
+
+                <!-- engine is also killed if oil level is critical. Minimum oil level in the sump for safe operation (according
+                to Lycomings manual, page 3-10) is 2 quarts, which explains the value of 1.97 below-->
                 <less-than>
                     <property>/engines/active-engine/oil-level</property>
                     <value>1.97</value>
                 </less-than>
-                
+
+                <!-- kill engine if the oil pressure is too low while the engine is running, according to Lycoming manaul 3-6 minimum oil pressure
+                in normal flight is 60psi, we use quite a bit below that to avoid excessive coughing during start. The engine will run for a while
+                probably at these low pressures, but if you have zero, you need to cut the engine fast! -->
+                <and>
+                    <property>/engines/active-engine/running</property>
+                    <equals>
```

Comment:
Why not replace this `<equals>` with `<not><property>/controls/switches/starter</property></not>` so that you don't need this `ready-oil-press-checker` hack?

Reply:
It sometimes kept coughing after the starter was disengaged, until the RPM, thus oil pressure had risen to a suitable value.

Reply:
What about `/engines/active-engine/cranking`?